orchagent/portsorch.cpp

```diff
@@ -1031,6 +1077,9 @@ bool PortsOrch::initPort(const string &alias, const set<int> &lane_set)
 
                 m_flexCounterTable->set(key, fields);
 
+    		PortUpdate update = {p, true };
+    		notify(SUBJECT_TYPE_PORT_CHANGE, static_cast<void *>(&update));
```

Comment:
what is the purpose to notify the port change here?

Reply:
INT sink port configuration and queue report configuration depend on port add/remove events. Dtelorch agent receives the configuration for sink ports and queue reports and keeps a copy of it locally. When the corresponding port is added/removed, the config is pushed/torn down.